src/commands/link/link.ts

```diff
@@ -214,8 +223,6 @@ or run ${chalk.cyanBright('netlify sites:create')} to create a site.`)
       }
       break
     }
-    default:
-      return
```

Comment:
This is not a breaking change, doubly so:
- this was an exhaustive switch-case
- the condition right below this `(if !site)` results in the same behaviour as before